plugins/decl_hdf5/dataset_op.h

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
  * Copyright (C) 2015-2024 Commissariat a l'energie atomique et aux energies alternatives (CEA)
- * Copyright (C) 2021 Institute of Bioorganic Chemistry Polish Academy of Science (PSNC)
+ * Copyright (C) 2021-2022 Institute of Bioorganic Chemistry Polish Academy of Science (PSNC)
```

Comment:
The year of this copyright, it is correct?  Do you need add some new copyright due to Yacine?

Reply:
This difference correspond to the proposition of @jbigot  in the gitlab repository

Reply:
Same as above